src/piece_of_cake.qos

```diff
@@ -15,8 +15,9 @@ QDISC=cake
 
 
 # to keep this as simple as possible we ignore the *_CAKE_OPTS from defaults.sh
-INGRESS_CAKE_OPTS="besteffort"
-EGRESS_CAKE_OPTS="besteffort"
+# and clear the DSCP bits from ingress & egress
+INGRESS_CAKE_OPTS="besteffort wash"
+EGRESS_CAKE_OPTS="besteffort wash"
```

Comment:
Should we really always wash when using a single-tier queue? I would at least gate it on the same setting as in layer_cake for egress...